pkg/webhook/server/server.go

```diff
@@ -282,16 +284,29 @@ func (s *Server) mutate(ctx context.Context, obj runtime.Object) (runtime.Object
 		return nil, errors.New("request is not of type apiextensions v1")
 	}
 	review.Response = s.MutationWebhook.Mutate(ctx, review.Request)
+	s.logAdmissionReview(review, "mutate")
+
 	return review, nil
 }
 
+func (s *Server) logAdmissionReview(review *admissionv1.AdmissionReview, operation string) {
+	if review.Request == nil {
+		s.log.Info(operation + " request handled nil request")
+	} else if review.Response == nil {
+		s.log.Info(fmt.Sprintf("%s request handled name %s namespace %s", operation, review.Request.Name, review.Request.Namespace))
+	} else {
+		s.log.Info(fmt.Sprintf("%s request handled name %s namespace %s response uuid %s allowed %t", operation, review.Request.Name, review.Request.Namespace, review.Response.UID, review.Response.Allowed))
```

Comment:
A couple things:

- Instead of `fmt.Sprintf` we should use key value pairs that can be passed to the `Info` method, i.e `Info("request received by validating webhook", namespace, review.Request.Namespace...`. This will make the log lines look more similar to what we log in other places in code.

- It would be good if we also logged what type of resource was being handled, you should be able to get that from `Request.Kind`.

- This message should be logged at debug rather than Info level as else we will be spamming the webhook logs